pkgs/games/chiaki4deck/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, mkDerivation
+, cmake
+, pkg-config
+, protobuf
+, python3Packages
+, ffmpeg_6
+, libopus
+, qtbase
+, qtmultimedia
+, qtsvg
+, SDL2
+, libevdev
+, udev
+, hidapi
+, fftw
+}:
+
+mkDerivation rec {
+  pname = "chiaki4deck";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "streetpea";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Pb+PuGl9KnYwnpcUJNTn29zpqEtYfPVxmzjRBYDymII=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    protobuf
+    python3Packages.protobuf
+    python3Packages.python
+    python3Packages.setuptools
```

We probably want to build a python env

```suggestion
    python3
    python3.pkgs.protobuf
    python3.pkgs.setuptools
```